modules/ROOT/pages/managing-api-clients.adoc

```diff
@@ -3,19 +3,37 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic provides an overview of client management on Anypoint Platform. As an API owner, you can apply an OAuth 2.0 policy to authenticate client applications that try to access your API. You need an OAuth provider to use an OAuth 2.0 policy. One of the following OAuth providers is recommended:
+This topic provides an overview of client management on Anypoint Platform. As an API owner, you can apply an OAuth 2.0 policy to authenticate client applications that try to access your API. You need an OAuth 2.0 provider to use an OAuth 2.0 policy.
```

Comment:
Avoid self-reference to "this topic/section/document." Try to reword.

Reply:
```suggestion
Anypoint Platform provides its own client provider service, but you can also configure other client providers to authorize client applications. As an API owner, you can apply an OAuth 2.0 policy to authorize client applications that try to access your API. You need an OAuth 2.0 provider to use an OAuth 2.0 policy.
```